jrportfolio.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Jason Rees Portfolio</title>	
+	<link href="normalize.css" rel="stylesheet" type="text/css">
```

Comment:
These links and meta tag should be inline with the title tag, as they aren't nested in anything.